src/safir/uws/_exceptions.py

```diff
@@ -238,7 +238,13 @@ def to_sentry(self) -> SentryEventInfo:
         if self.job_id:
             info.tags["job_id"] = self.job_id
         if self._detail:
-            info.contexts.setdefault("info", {})["detail"] = self._detail
+            info.contexts.setdefault("original_exception", {})["detail"] = (
+                self._detail
+            )
+        if self._cause_type:
+            info.contexts.setdefault("original_exception", {})[
+                "cause_type"
+            ] = self._cause_type
```

Comment:
```suggestion
        if self._detail or self._cause_type:
            context = info.contexts.setdefault("original_exception", {})
            if self._detail:
                context["detail"] = self._detail
            if self._cause_type:
                context["cause_type"] = self._cause_type
```
is a little more succinct since it avoids the long lines that create awkward formatting.